pkg/agent/openflow/pipeline_windows.go

```diff
@@ -281,5 +281,24 @@ func (c *client) hostBridgeUplinkFlows(localSubnet net.IPNet, category cookie.Ca
 			Cookie(c.cookieAllocator.Request(category).Raw()).
 			Done(),
 	}
+	if c.encapMode.SupportsNoEncap() {
+		// If NoEncap is enabled, the return traffic goes the symmetric path as the requests. After inputting from uplink,
+		// the return traffic is forwarded to host stack via br-int. Then it is routed to antrea-gw0 and enters OVS again.
+		// Finally, it reaches its destination.
+		flows = append(flows, c.pipeline[conntrackStateTable].BuildFlow(priorityHigh).MatchProtocol(binding.ProtocolIP).
+			MatchRegRange(int(marksReg), markTrafficFromUplink, binding.Range{0, 15}).
+			MatchCTStateNew(true).MatchCTStateTrk(true).
+			MatchDstIPNet(localSubnet).
+			Action().LoadRegRange(int(marksReg), macRewriteMark, macRewriteMarkRange).
+			Action().ResubmitToTable(EgressRuleTable).
```

Comment:
Why we need to make ingress connection go egress table?

Reply:
Now I removed this flow table. This one was by old design to let return traffic go to destination Pod directly. I re-tested and this one seems not needed any more.